tkeysign.go

```diff
@@ -268,3 +270,37 @@ func (s Signer) getSig() ([]byte, error) {
 	// ed25519 signature
 	return rx[3 : 3+64], nil
 }
+
+// SignPh signs the message in data and returns an ed25519 signature.
```

Comment:
Please add an explanation that SignPh takes a SHA-512 hash

Reply:
Fixed.